GFRave/include/GFRaveTrackParameters.h

```diff
@@ -53,8 +53,8 @@ class GFRaveTrackParameters : public TObject
   public:
     // constructors, destructors
     GFRaveTrackParameters();
-    GFRaveTrackParameters(const Track* track, MeasuredStateOnPlane* originalState, double weight, const TVectorD & state6, const TMatrixDSym & cov6x6, bool isSmoothed);
-    GFRaveTrackParameters(const Track* track, MeasuredStateOnPlane* originalState, double weight);
+    GFRaveTrackParameters(const Track* track, double weight, const TVectorD & state6, const TMatrixDSym & cov6x6, bool isSmoothed);
```

Comment:
Doesn't that break a lot of things, e.g. in basf2?

Reply:
I agree, change of interface should be avoided. What you can do is write down the unused arguments like `(..., MeasuredStateOnPlane* /* originalState */, ...). This will avoid the unused argument compiler warning but keep the interface.